### ISSUE TITLE
Bad behavior while updating source texts

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-android#1926]
Notes
======
I'm in the source selection dialog in a Philippians project trying to download the USB by clicking the circular arrow

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 156
UDID | 3c7d0770c6265875
Android Release | 5.1.1
Android SDK | 22
Brand | google
Device | flo

**Comment by cckozie** [comment]
I tried twice to download the update to the UDB. The first time I missed the popup message, but the circular arrow stayed there so I clicked it again. The second time I saw that the message indicated a failure and the circular arrow stayed there again. I closed and reopened the project, and not the circular arrow is not displayed by the udb.

The rest of the story... 
This project was imported from a usfm file so when I first opened it I was presented with the source selection menu. I saw that the ulb had an update available, so I clicked on its circular arrow to download it. When the download finished, the circular arrow was gone, but the ulb had not been moved to the Selected group as I expected. I clicked on the checkbox to select it and then clicked Confirm.

Once in the project I wanted to test to see if source languages are actually not automatically selected after the user has chosen to update them on the source selection screen, so I went back to that dialog. I clicked on the circular arrow for another source language. After the download completed, it was automatically selected. I clicked Confirm to go back into the project.

I then went back to the source selection screen and deselected both source languages. This is the point at which I appeared to fail when downloading the udb update.

**Comment by cckozie** [comment]
After all of that above, I deleted the php project, uninstalled the app, and renamed the resource_containers folder. I then reinstalled the app, imported the usfm file, clicked the ulb circular arrow to update the source, and it worked expected moving it to the Selected group when the download completed. I then unselected the ulb and clicked on the circular arrow for the udb. It all worked as expected so I'm going to throw this issue back into the issue backlog lane for now.
